action.yml

```diff
@@ -23,6 +23,14 @@ inputs:
     description: 'A specific repo to sync the secrets to'
     mandatory: False
     default: ''
+  dependabotSecretsSync:
+    description: 'Sync secrets to dependabot (yes/no), default yes'
+    mandatory: False
+    default: 'no'
+  repo_owner:
+    description: 'Repository Owner Name'
+    mandatory: False
+    default: 'philips-internal'
```

Comment:
I would not use this as a default... the value should be passed

Reply:
The action will take the repo owner name as input from the calling workflow, As I needed default to be a non null value, I  assigned its value as philips-internal.

Reply:
So this -  the value should be passed

I dont understand clearly.

Reply:
Ok in that case, I will make this mandatory and hence remove the default param

Reply:
default - "org-name"